.github/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Doesn't it need `// +build tools` as the other similar files have it?

Reply:
I tested and looks like it does not need the `// +build tools` tag. However the package declaration is needle - otherwise the vendor from another project fails with 
```
github.com/gardener/gardener-extension-provider-aws/hack imports
	github.com/gardener/gardener/.github: module github.com/gardener/gardener@latest found (v0.0.0-00010101000000-000000000000, $HOME/go/src/github.com/gardener/gardener), but does not contain package github.com/gardener/gardener/.github
```

So I guess we can remove `// +build tools` from extensions/hack/tools.go, hack/.ci/doc.go, hack/api-reference/template/tools.go, hack/local-development/common/tools.go.